src/directives/ng-file-drop.ts

```diff
@@ -122,6 +122,16 @@ export class NgFileDropDirective implements OnChanges, OnInit {
       });
     }
 
+    if (this.options.maxSize > 0) {
+      this.files = this.files.filter(f => {
```

Comment:
In this case, I copied the code above, for extension filter.
I'll change both of them.